File: Api.Server/src/Services/Center/Application/Dto/QueryApplicationPagedInput.cs
namespace Fast.Center.Service.Application.Dto;

/// <summary>
/// <see cref="QueryApplicationPagedInput"/> 获取应用分页列表输入
/// </summary>
public class QueryApplicationPagedInput : PagedInput
{
    /// <summary>
    /// 版本
    /// </summary>
    public EditionEnum? Edition { get; set; }
}

File: Web.Admin/src/api/services/application/models/QueryApplicationPagedInput.ts
import { PagedInput } from "fast-element-plus";
import { EditionEnum } from "@/api/enums/EditionEnum";

/**
 * Fast.Center.Service.Application.Dto.QueryApplicationPagedInput 获取应用分页列表输入
 */
export interface QueryApplicationPagedInput extends PagedInput  {
  /**
   * 
   */
  edition?: EditionEnum;
  /**
   * 租户Id
   */
  tenantId?: number;
}

